### DISCUSSION TITLE
Any Plans to maintain tui-textarea?

### DISCUSSION THREAD

**Discussion by AmmarAbouZor** [ratatui/ratatui#507] [Discussion]
Hi there,

The crate [tui-textarea](https://github.com/rhysd/tui-textarea) seems to be unmaintained anymore. We've had open issues and pull request there for about 6 months without a response from the maintainer. I think he had a lot of great project and maybe he doesn't have time for it or he has lost interest.
I think the crate is really great and I wish we can save it, therefore I want to ask if there are any plans for the crate here, maybe contact the maintainer and offer him some help or fork the project and keep maintaining it? 
I think the crate is by far the most complete text-input widget for rata-tui and would really feel sorry if such a great work stays unusable with the ratatui

**Comment by AmmarAbouZor** [comment]
It seems that the crate is still maintained. A new release has arrived and it supports ratatui.
Time to migrate :tada:

**Comment by joshka** [comment]
Hi yup - sorry for no response on this issue. I contacted the tui-textarea privately about this. It's good to see some movement there.

**Reply by AmmarAbouZor** [reply]
Thank you so much! I have been waiting on that for months and started to lose hope since the maintainer seems to be busy with a lot of other huge projects

**Reply by joshka** [reply]
Yeah, that's one of those eternal problems with single person open source projects. Time is finite.
